packages/builder/src/builderStore/api.js

```diff
@@ -21,10 +21,20 @@ export const patch = apiCall("PATCH")
 export const del = apiCall("DELETE")
 export const put = apiCall("PUT")
 
-export default {
-  post,
-  get,
-  patch,
-  delete: del,
-  put,
-}
+// usage: api(instanceId).post(...) ... will supply instance Id in header
+const api = instanceId => ({
+  post: apiCall("POST", instanceId),
+  get: apiCall("GET", instanceId),
+  patch: apiCall("PATCH", instanceId),
+  delete: apiCall("DELETE", instanceId),
+  put: apiCall("PUT", instanceId),
+})
+
+// usage: api.post(...)... will not supply instanceid in header
```

Comment:
It's confusing to overload the `api` module just for the sake of one argument. It's better to make this one module where you could possible just pass down custom headers/options as another argument

Reply:
yea - i have done other things since making this change, so i can fix this ok now i think